android/logto-client/src/test/kotlin/io/logto/sdk/android/LogtoClientTest.kt

```diff
@@ -14,6 +18,191 @@ import org.junit.Test
 
 class LogtoClientTest {
     private val oidcConfigResponseMock: OidcConfigResponse = mockk()
+    private val logtoConfigMock: LogtoConfig = mockk()
+    private lateinit var logtoClient: LogtoClient
+
+    companion object {
+        private const val TEST_SCOPE_1 = "scope_1"
+        private const val TEST_SCOPE_2 = "scope_2"
+        private const val TEST_SCOPE_3 = "scope_3"
+
+        private const val TEST_RESOURCE_1 = "resource_1"
+        private const val TEST_RESOURCE_2 = "resource_2"
+        private const val TEST_RESOURCE_3 = "resource_3"
+
+        private const val TEST_CLIENT_ID = "client_id"
+        private const val TEST_REFRESH_TOKEN = "refreshToken"
+        private const val TEST_TOKEN_ENDPOINT = "tokenEndpoint"
+        private const val TEST_ACCESS_TOKEN = "accessToken"
+        private const val TEST_ID_TOKEN = "idToken"
+        private const val TEST_EXPIRE_IN = 60L
+    }
+
+    @Test
+    fun `getAccessToken should failed without authenticated`() {
```

Comment:
nit: should `fail` without `being` authenticated

Reply:
updated